[2025-05-22T07:27:59.073+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: log_etl_pipeline.load_to_db scheduled__2025-05-21T00:00:00+00:00 [queued]>
[2025-05-22T07:27:59.093+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: log_etl_pipeline.load_to_db scheduled__2025-05-21T00:00:00+00:00 [queued]>
[2025-05-22T07:27:59.094+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-22T07:27:59.112+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_to_db> on 2025-05-21 00:00:00+00:00
[2025-05-22T07:27:59.116+0000] {standard_task_runner.py:60} INFO - Started process 692 to run task
[2025-05-22T07:27:59.120+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'log_etl_pipeline', 'load_to_db', 'scheduled__2025-05-21T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/log_etl_dag.py', '--cfg-path', '/tmp/tmplfgcmywd']
[2025-05-22T07:27:59.123+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask load_to_db
[2025-05-22T07:27:59.183+0000] {task_command.py:423} INFO - Running <TaskInstance: log_etl_pipeline.load_to_db scheduled__2025-05-21T00:00:00+00:00 [running]> on host 359b0195ed38
[2025-05-22T07:27:59.278+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='log_etl_pipeline' AIRFLOW_CTX_TASK_ID='load_to_db' AIRFLOW_CTX_EXECUTION_DATE='2025-05-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-21T00:00:00+00:00'
[2025-05-22T07:27:59.279+0000] {logging_mixin.py:188} INFO - Loading data into PostgreSQL...
[2025-05-22T07:27:59.280+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-22T07:27:59.292+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=log_etl_pipeline, task_id=load_to_db, execution_date=20250521T000000, start_date=20250522T072759, end_date=20250522T072759
[2025-05-22T07:27:59.341+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-22T07:27:59.369+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
